### PR TITLE
update json responses to add space after `:` + fix typo

### LINKED ISSUE #1596: Update json responses
Update json responses to add a space after each colon `:`
### Example:
Update:
```
"uid":1,
```
to 
```
"uid": 1,
```

Currently we have a mix of with and without space

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1596 

Example:

Update:
```
"uid":1,
```
to
```
"uid": 1,
```

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -331,7 +331,7 @@ filtering_guide_2: |-
   curl \
     -X POST 'http://localhost:7700/indexes/movies/search' \
     -H 'Content-Type: application/json' \
-    --data-binary '{ "q":"Batman",  "filter": "release_date > 795484800 AND (director = \"Tim Burton\" OR director = \"Christopher Nolan\")" }'
+    --data-binary '{ "q": "Batman",  "filter": "release_date > 795484800 AND (director = \"Tim Burton\" OR director = \"Christopher Nolan\")" }'
```
Comment (dichotommy) [inline .code-samples.meilisearch.yaml:334]:
```suggestion
    --data-binary '{ "q": "Batman", "filter": "release_date > 795484800 AND (director = \"Tim Burton\" OR director = \"Christopher Nolan\")" }'
```

**Review by dichotommy — DISMISSED** [review]
LGTM 🥽
Did you do this by hand??

**Comment by maryamsulemani97** [comment]
>Did you do this by hand??

Regex + find and replace 🤫

**Review by dichotommy — APPROVED** [review]
LGTM 🪂

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**